pytest/data_handling/postprocessors/test_postprocessors.py

```diff
@@ -1,6 +1,63 @@
 import pytest
 from typing import Tuple
-from ml_gym.data_handling.postprocessors.postprocessor import OneHotEncodedTargetPostProcessor, LabelMapperPostProcessor
+
+from data_stack.dataset.iterator import InformedDatasetIterator
+from data_stack.dataset.meta import IteratorMeta, DatasetMeta
+from ml_gym.data_handling.postprocessors.feature_encoder import CategoricalEncoder
+from ml_gym.data_handling.postprocessors.postprocessor import OneHotEncodedTargetPostProcessor, \
+    LabelMapperPostProcessor, FeatureEncoderPostProcessor
+
+from mocked_class import MockedIterator
+
+
+class TestFeatureEncoderPostProcessor:
+    @pytest.fixture
+    def iterators(self):
+        iterator = MockedIterator()
+        iterator_meta = IteratorMeta(sample_pos=0, target_pos=1, tag_pos=2)
+        dataset_meta = DatasetMeta(identifier="identifier", dataset_name="dataset_name",
+                                   dataset_tag="dataset_tag", iterator_meta=iterator_meta)
+        iterator = InformedDatasetIterator(iterator, dataset_meta)
+        iterators = {"train": iterator, "test": iterator}
+        return iterators
+
+    @pytest.fixture
+    def sample(self, iterators):
+        return iterators["train"][0]
+
+    @pytest.fixture
+    def sample_position(self, iterators):
+        sample_position = iterators["train"].dataset_meta.sample_pos
+        return sample_position
+
+    @pytest.fixture
+    def custom_encoders(self):
+        custom_encoders = {"categorical2": CategoricalEncoder}
+        return custom_encoders
+
+    @pytest.mark.parametrize('sequential', [True, False])
+    def test_postprocess(self, iterators, sample_position, sample, custom_encoders, sequential):
+        feature_encoding_configs = [{"feature_type": "categorical", "feature_names": [0, 1, 2], "train_split": "train"}]
```

Comment:
we should also test for the "continuuous" feature type

Reply:
done